GO_Sync/src/main/java/edu/usf/cutr/go_sync/io/GTFSReadIn.java

```diff
@@ -136,6 +136,9 @@ public String readAgency(String agency_fName)
             { 
                 if (isFirstLine) {
                     isFirstLine = false;
+                    if (thisLine.startsWith("\uFEFF")) {
```

Comment:
Instead of repeating "\uFEFF", could you please define this as final static String constant at top of class?